src/main/java/com/codisimus/plugins/phatloots/PhatLootsAPI.java

```diff
@@ -141,6 +143,15 @@ public static boolean loot(Block block, Player player, boolean autoSpill) {
 
         if (flagToBreak) {
             plChest.breakChest(player, plChest.getResetTime(phatLoots));
+        } else {
+            ChestOpenEvent event = new ChestOpenEvent(plChest,
+                    player.getOpenInventory().getTopInventory(),
+                    player);
+            if (event.getInventory().getType() == InventoryType.PLAYER)
+                return true;
```

Comment:
```suggestion
        } else {
            if (event.getInventory().getType() == InventoryType.PLAYER) {
                return true;
            }
            ChestOpenEvent event = new ChestOpenEvent(plChest,
                    player.getOpenInventory().getTopInventory(),
                    player);
```
Small nitpick - if we are not going to call the event, there's no reason to create an instance of it.

Reply:
Done, I cached the inventory since event doesn't exist yet